cmd/alias.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/Originate/git-town/lib/git"
+	"github.com/Originate/git-town/lib/script"
+	"github.com/spf13/cobra"
+)
+
+var aliasCommand = &cobra.Command{
+	Use:   "alias (true | false)",
+	Short: "Adds or removes default global aliases",
+	Run: func(cmd *cobra.Command, args []string) {
+		addOrRemoveAliases(convertStringToBool(args[0]))
+	},
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		err := validateArgsCount(args, 1)
+		if err != nil {
+			return err
+		}
+		return validateBooleanArgument(args[0])
+	},
+}
+
+func addOrRemoveAlias(command string, toggle bool) {
+	key := "alias." + command
+	value := "!gt " + command
+	if toggle {
```

Comment:
This method does two very different things. I propose to split it up into two methods `addAlias` and `removeAlias`. Then move the if-clause to the `Run` method. Evaluating what to do based on command-line parameters is a part of the controller responsibility. Implementing how it is done is part of the business logic.